README.md

```diff
@@ -86,9 +82,9 @@ Detailed examples of using SMIRNOFF with the toolkit can be found [in the docume
 
 See [`FAQ.md`](FAQ.md) for answers to a variety of common problems, such as:
 * Why do I need to provide molecules corresponding to the components of my system, or a `Topology` with bond orders?
-* Can I use an AMBER, CHARMM, or gromacs topology/coordinate file as a starting point for applying a SMIRNOFF force field?
+* Can I use an Amber, CHARMM, or GROMACS topology/coordinate file as a starting point for applying a SMIRNOFF force field?
 * What if I am starting from a PDB file?
 
 # Contributors
 
-For a full list of contributors, see the [GitHub Contributors page](https://github.com/openforcefield/openff-toolkit/graphs/contributors).
+For a partial list of contributors, see the [GitHub Contributors page](https://github.com/openforcefield/openff-toolkit/graphs/contributors). Others whose work constitutes significant contributions but did not make it into the `git` history include Shuzhe Wang.
```

Comment:
Resolves #257 - surely there are others who could be listed, but including one person is better than including none.